spec/samples/inline_trackers.js

```diff
@@ -170,6 +170,18 @@ export const inlineTrackersParsed = {
             midpoint: ['http://example.com/linear-midpoint'],
             complete: ['http://example.com/linear-complete'],
             start: ['http://example.com/linear-start'],
+            loaded: ['http://example.com/linear-loaded'],
+            complete: ['http://example.com/linear-complete'],
```

Comment:
Thank you for your feedback :)
however, this shouldn't be an issue because the request is made via an image, which avoids both mixed content and CORS errors.